### ISSUE TITLE
running as non root & unistalling

### ISSUE THREAD

**Issue by Nihim** [TurboGit/hubicfuse#116]
OS: Debian 8 64

If I made an error kindly point it out to me! So I installed hubicfuse, made the config and authenticated with root on a vps cause I was testing something. Then I decided to run it from non root. I copied the config to the user home and chown it.

usermod complained that fuse group didn't exist so I created it.
hubicfuse /mnt/hubic -o noauto_cache,sync_read,allow_other throws: "fuse: failed to exec fusermount: No such file or directory"

**What I am trying to do is the mounted hubic dir and all subdirs to be owned by non root.**

Also tried on the /root/.hubicfuse to add
enable_chmod=true
enable_chown=true

and chown -R the dir it stuck there doing nothing (after few min not even 1 file had been modifed) not sure if that is to be expected due to the amount of files.

Every time I changed the config I unmounted and remounted the config as I am not sure if that is needed or not.

so I would appreciate it if someone explained what I am doing wrong.

Also would it be possible to support make unistall?

**Comment by romanrm** [comment]
> fuse group didn't exist

> failed to exec fusermount: No such file or directory

Do you have the 'fuse' package installed properly in your system?
Which OS?
In Debian you should **apt-get install fuse** and then have **/bin/fusermount**.

**Comment by Nihim** [comment]
Nope, I did not. Thanks!

Since I have this open is it possible to sync without deleting from the target? So basically only new files are added to hubic and if any have been removed from the source, hubic does not remove them. I saw no flag like that sadly.

**Comment by romanrm** [comment]
> is it possible to sync without deleting from the target? So basically only new files are added to hubic and if any have been removed from the source, hubic does not remove them

I don't know what do you use to 'sync', but the **rsync** program can certainly do that. See **man rsync** for the variants of the **--delete** switch. Also please don't use the same report for multiple unrelated questions in the future.

**Comment by dan-cristian** [comment]
I suggest keeping the config options below to false, as they are meant to limit access to hubic files based on the user / file rights.
enable_chmod=false
enable_chown=false

**Comment by Nihim** [comment]
> I don't know what do you use to 'sync'...

Yea I guess it was late, the sync function is from another fuse application.
Forgot that since it's locally mounted I can mv / cp / local rsync or anything else on it.

> I suggest keeping the config options below to false, as they are meant to limit access to hubic files based on the user / file rights.

Since I corrected my mistake with romanrm's help I removed those options.

> Also please don't use the same report for multiple unrelated questions in the future.

Noted. I am guessing you are referring though to the sync question and not on the initial post where I described my problem and also asked about the unistall of hubicfuse.

Thank you both for your help.
